### PR TITLE
restored earlier version

### PR DESCRIPTION
**Pull request by MariePMcGarry**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/google/workspace/collect-logs.md [hunk 1]
```diff
@@ -18,19 +18,21 @@ This procedure explains how to collect logs from Google Workspace and ingest the
...
     * Google Drive
     * Google Login
     * Google Token
 
 ### About Source Configuration  
 
-Configure one [Google Workspace Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source) for each Google App from which you want to collect events:
+Currently, the source name for Google Workspace is still **G Suite Apps Audit Source**, which will be changed/updated shortly.
+
+Configure one [G Suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source) for each Google App from which you want to collect events:
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:35]: Change to "Google Workspace Apps Audit Source"?

File: docs/integrations/google/workspace/collect-logs.md [hunk 2]
```diff
@@ -18,19 +18,21 @@ This procedure explains how to collect logs from Google Workspace and ingest the
 **Google Workspace Alert Center** alerts are in JSON format. Most of the alerts have a few common fields. The differences are in the data section of the JSON where the alert type specific details are recorded. For more information, see this Google Workspace [Alert document](https://developers.google.com/admin-sdk/alertcenter/reference/alert-types).
 
 
-## Configure Collection for Google Workspace Source
+## Configure Collection for Google Workspace Audit Source
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:21]: "Workspace Apps Audit" (add "Apps")

File: docs/integrations/google/workspace/collect-logs.md [hunk 3]
```diff
@@ -65,28 +67,28 @@ Only one Source should be configured per app. In other words, you might set up o
...
 
-When creating or modifying a Google Source, you will be required to authenticate with Google using the credentials of a user that has access rights to the account, and to the Reports API. See Google's [Reports API: Prerequisites](https://developers.google.com/admin-sdk/reports/v1/guides/prerequisites) documentation for more details. During Google's OAuth consent flow, you will also be asked to grant the Sumo Logic app permission to use the Reports API.
+When creating or modifying a Google Apps Audit Source, you will be required to authenticate with Google using the credentials of a user that has access rights to the account, and to the Reports API. See Google's [Reports API: Prerequisites](https://developers.google.com/admin-sdk/reports/v1/guides/prerequisites) documentation for more details. During Google's OAuth consent flow, you will also be asked to grant the Sumo Logic app permission to use the Reports API.
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:70]: "or modifying a Google Workspace Apps Audit Source"

File: docs/integrations/google/workspace/collect-logs.md [hunk 4]
```diff
@@ -198,3 +200,496 @@ _sourceCategory=google*
 | count by email
 | sort by _count desc, email asc
 ```
+
+
+## Configure Collection for Google Workspace Alert Center
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:205]: This section is for the legacy AlertCenter collection so we should remove it and replace with the new C2C Google Workspace AlertCenter one.

**Review by duchatran — CHANGES_REQUESTED** [review]
Added comments

File: docs/integrations/google/workspace/collect-logs.md (hunk 1 above)
Comment (MariePMcGarry) [reply docs/integrations/google/workspace/collect-logs.md:35]: @duchatran Please be specific. There are several references to sources in lines 33 - 35. Please confirm which to change to "Google Workspace Apps Audit Source". Also, do any of the paths in links needs updating?

File: docs/integrations/google/workspace/collect-logs.md (hunk 3 above)
Comment (MariePMcGarry) [reply docs/integrations/google/workspace/collect-logs.md:70]: Updated file.

File: docs/integrations/google/workspace/collect-logs.md [hunk 5]
```diff
@@ -65,28 +67,28 @@ Only one Source should be configured per app. In other words, you might set up o
...
 
 :::note
-Authentication must be with a new Google Workspace Source. We do not support re-authenticating existing sources.
+Authentication must be with a new Google Workspace Apps Audit Source. We do not support re-authenticating existing sources.
 :::
 
 ### Configure a Collector
 
-Configure a [Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector) for the Google Workspace Sources you'll set up below.
+Configure a [Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector) for Google Apps Audit.
```
[the remark this answers is not in the document]
Comment (MariePMcGarry) [reply docs/integrations/google/workspace/collect-logs.md:78]: updated file in PR.

File: docs/integrations/google/workspace/collect-logs.md (hunk 4 above)
Comment (MariePMcGarry) [reply docs/integrations/google/workspace/collect-logs.md:205]: Fixed in file in PR.

File: docs/integrations/google/workspace/collect-logs.md (hunk 1 above)
Comment (duchatran) [reply docs/integrations/google/workspace/collect-logs.md:35]: Just change the text "G Suite Apps Audit Source".

**Review by duchatran — APPROVED** [review]
LGTM

**Review by duchatran — APPROVED** [review]
LGTM

**Review by stacykor — APPROVED** [review]

**Pull request merged**